scripts/build/concatenate-javascript.sh

```diff
@@ -48,3 +52,8 @@ if [ -z "$ARGV_ENTRY_POINT" ] || [ -z "$ARGV_OUTPUT" ]; then
 fi
 
 "$BROWSERIFY" "$ARGV_ENTRY_POINT" --node --outfile "$ARGV_OUTPUT"
+
+if [ "$ARGV_MINIFY" == "true" ]; then
+  uglifyjs --compress --output "$ARGV_OUTPUT.MIN" -- "$ARGV_OUTPUT"
+  mv "$ARGV_OUTPUT.MIN" "$ARGV_OUTPUT"
```

Comment:
Not sure I follow. Do you mean that we should check if `$ARGV_OUTPUT.MIN` before moving it?